local/include/core/http2.h

```diff
@@ -197,12 +200,14 @@ class H2Session : public TLSSession
   /** Populate an nghttp2 vector from the information in an HttpHeader instance.
    *
    * @param[in] hdr The instance from which to pack headers.
+   * @param[in] is_trailer Whether the headers are trailers.
    * @param[out] nv_hdr The packed headers.
    * @param[out] hdr_count The size of the nv_hdr vector.
    *
    * @return Any errata information from the packing operation.
    */
-  swoc::Errata pack_headers(HttpHeader const &hdr, nghttp2_nv *&nv_hdr, int &hdr_count);
+  swoc::Errata
+  pack_headers(HttpHeader const &hdr, bool is_trailer, nghttp2_nv *&nv_hdr, int &hdr_count);
```

Comment:
Please add a `constexpr bool IS_TRAILER = true;` and use that in the call sites:

This:
```
pack_headers(hdr, !IS_TRAILER, hdrs, hdr_count);
```

Is clearer than:
```
pack_headers(hdr, false, hdrs, hdr_count);
```

Reply:
Good point. Much cleaner.